### PR TITLE
add star rating sort to grid view

### PR DESCRIPTION
**Pull request by JannikHoelling**
**Changes**
Added sorting by star rating(called CommunityRating in the jellyfin-api) to the sortOption HashMap
Tested it in the android studio emulator and it worked.

**Issues**
Fixes https://github.com/jellyfin/jellyfin-androidtv/issues/1135

### CODE REVIEW

File: app/src/main/java/org/jellyfin/androidtv/ui/GridFragment.java [hunk 1]
```diff
@@ -152,8 +152,9 @@ public SortOption(String name, String value, SortOrder order) {
         sortOptions.put(1, new SortOption(TvApp.getApplication().getString(R.string.lbl_date_added), "DateCreated,SortName", SortOrder.Descending));
         sortOptions.put(2, new SortOption(TvApp.getApplication().getString(R.string.lbl_premier_date), "PremiereDate,SortName", SortOrder.Descending));
         sortOptions.put(3,new SortOption(TvApp.getApplication().getString(R.string.lbl_rating), "OfficialRating,SortName", SortOrder.Ascending));
-        sortOptions.put(4,new SortOption(TvApp.getApplication().getString(R.string.lbl_critic_rating), "CriticRating,SortName", SortOrder.Descending));
-        sortOptions.put(5,new SortOption(TvApp.getApplication().getString(R.string.lbl_last_played), "DatePlayed,SortName", SortOrder.Descending));
+        sortOptions.put(4,new SortOption(TvApp.getApplication().getString(R.string.lbl_stars), "CommunityRating,SortName", SortOrder.Descending));
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/ui/GridFragment.java:155]: I'm not sure about the used label, We might want to use "Community Rating" or something like that as label.

Comment (JannikHoelling) [reply app/src/main/java/org/jellyfin/androidtv/ui/GridFragment.java:155]: I can add a new string called lbl_community_rating. To make it consistent i can also change the enum RatingType to those labels. There RATING_TOMATOES with lbl_tomatoes and RATING_STARS with lbl_stars is still used. 
If that would be ok i'm gonna do that.

The jellyfin webui is currently using IMDb Rating and Critic Rating as labels. That would also be an option.

Comment (thornbill) [reply app/src/main/java/org/jellyfin/androidtv/ui/GridFragment.java:155]: I would recommend using the same labels as web for consistency.

Comment (nielsvanvelzen) [reply app/src/main/java/org/jellyfin/androidtv/ui/GridFragment.java]: I think we should not use the imdb name as it's trademarked, same reason that we don't use it when choosing which rating to show on cards.
Besides, the value is not always an imdb rating. different metadata provider plugins can put different values in it.

Comment (JannikHoelling) [reply app/src/main/java/org/jellyfin/androidtv/ui/GridFragment.java]: Should we also rename the option in the settings to Critic and Community Rating? Would make it equal everywhere.

**Review by nielsvanvelzen — APPROVED** [review]

**Review by thornbill — APPROVED** [review]

**Pull request merged**